cubism-core-sys/build.rs

```diff
@@ -29,6 +31,17 @@ fn main() {
             lib_dir.push("140");
         },
         ("apple", "darwin") => {
+            if cfg!(feature = "dylib") {
```

Comment:
This should be 
```rust
if !cfg!(feature = "static-link") {
```
or
```rust
if linking_strategy != "static" {
```

Reply:
Oh, It won't work because there is no "dylib" feature now.

I'd prefer
```rust
if linking_strategy == "dylib" {
```
but `linking_strategy != "static"` has smaller comparing cost so I'll choose this.